docs/getting-started/README.md

```diff
@@ -1 +1,7 @@
-# getting-started/README.md
+# What is a CRM
+
+A Customer Relationship Management, well-known as CRM, is a way to track relationships with customers. Typically, this is done by storing data, such as contact information and other insights about a customer (recent emails, meetings, phone calls, etc.). CRM’s can be found in many forms, but at Cal-ITP, we use a platform named [Hubspot](https://www.hubspot.com/) for our CRM. Within Hubspot are multiple products that allow us to build out our CRM: *service, sales and marketing*. These products and our context for usage are detailed in this guide.
```

Comment:
> A Customer Relationship Management, well-known as CRM,

Edit: A Customer Relationship Management software, well-known as a CRM, 

> we use a platform named [Hubspot]

Edit:...we use a platform called [Hubspot]. OR ...we use [Hubspot] 
("named" sounds odd to me...am I being too picky...)

> These products and our context for usage are detailed in this guide.

Edit: These products, and our context for using them, are detailed in this guide.